### ISSUE TITLE
Command Line Overview documentation contains broken links to GATK site

### ISSUE THREAD

**Issue by gbrandt6** [broadinstitute/picard#1664]
### Description 
There are multiple links that are broken in the Picard Command Line Overview and are leading to old GATK website pages. One example of this issue is in the docs for [CollectInsertSizeMetrics](https://broadinstitute.github.io/picard/command-line-overview.html#CollectInsertSizeMetrics). There is a link to the GATK Dictionary which goes to https://www.broadinstitute.org/gatk/guide/article?id=6327 but this location no longer exists, it most likely happened when we changed the GATK website. We are not able to fix this issue by redirecting these links, so they will need to be updated in the Picard docs.

----

**Comment by jessicaway** [comment]
@gbrandt6 thanks for calling attention to this problem! 

Unfortunately the out of date Picard documentation is a known issue without a simple solution. More up to date documentation for Picard is available in the GATK documentation. For instance, I would recommend users view the [GATK documentation for Picard CollectInsertSizeMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360041414992-CollectInsertSizeMetrics-Picard-) rather than the [Picard Documentation for CollectInsertSizeMetrics](https://broadinstitute.github.io/picard/command-line-overview.html#CollectInsertSizeMetrics). 

I agree that having out of date documentation is confusing and problematic. I will leave this issue open until it is resolved

**Comment by gbrandt6** [comment]
Hi Jessica! Thanks for the response. Is there a plan in place to move any of the Picard info that is only available in the Picard documentation to the GATK documentation? For example, in the GATK docs, it links to the Picard docs here:

> Please see [InsertSizeMetrics](https://broadinstitute.github.io/picard/picard-metric-definitions.html#InsertSizeMetrics) for detailed explanations of each metric.

**Comment by jessicaway** [comment]
Yes, this is also a known issue. I think the plan is to eventually fix the picard documentation so that it can be published again (including the metrics information). 

The pipeline development team has issues in our backlog to address this when we have bandwidth, but I don't believe we have a timeline for that work. We would be very happy to assist if another team would like to take this on!
